README.md

```diff
@@ -127,41 +127,53 @@ You may also want to add a translation for the component and that can be found b
 ```
 "my-character-count": {
      ...
-     "maxlength": "You can only enter up to {{maxlength}} characters" - required by default
-
      (The keys bellow will allow translation of the hint text. %{count} is parsed by gds to show dynamic count)
 
      "textareaDescriptionText": "Enter up to %{count} characters" - shown, instead of dynamic count, to the user if javascript is disabled,
+     
      "charactersUnderLimitText": {
         "one": "you have one char left" - shown when user has one characters left
         "other": "you have %{count} characters left" - shown when user has n characters left
 
       } - shown to user when they have n characters remaining
+
       "charactersAtLimitText": "you have 0 characters remaining" - shown when user has no characters left
+
       "charactersOverLimitText": {
         "one": "you have entered 1 character too many " - shown when user has one character over the limit
         "other": "you have %{count} characters too many" - shown when user has n. characters over the limit
 
-     } - shown to user when they have exceed number of allowed characters     
+     } - shown to user when they have exceed number of allowed characters
+
+     validation: {
+        ...
+        "maxlength": "You can only enter up to {{maxlength}} characters" - required by default
```

Comment:
Thinking we bold the `required by default` text here and in the `maxwords` validation just so it jumps out to users that it's required